### ISSUE TITLE
PMMP VERY SLOW ! GENISYSPRO FASTER GURUN !

### ISSUE THREAD

**Issue by bettytwin** [NiclasOlofsson/MiNET#275]
Yes  !

pmmp has be slow .

pmmp:
-10 players 7% load 20tps

genisyspro:
- 10 players 43% load 13.96tps

, this mean pmmp not utilizing full cpu capacity !

please gurun , help us imrpve genisyyspro speed even more ?

**Comment by NiclasOlofsson** [comment]
Hmm, usually people would interpret that as "genisyspro is using more CPU than PMMP". But I'm no expert so I can't really say. I need more facts ands stats to be able to tell.

**Comment by bettytwin** [comment]
@NiclasOlofsson yes exactly !utilizess ALL the CPU to run faster ! (pmmp only utilized 7%, slow)

**Comment by NiclasOlofsson** [comment]
Ah yeah, i see how you mean. I guess my old car that used to use a lot of fuel was actually much better than my current care (that use considerable less). It simply doesn't use as much fuel which problem makes it go .. not as effecient.

**Comment by bettytwin** [comment]
i not have car , cant relate
